src/modules/roc_core/target_stdio/roc_core/log.cpp

```diff
@@ -77,6 +78,11 @@ void Logger::print(const char* module, LogLevel level, const char* format, ...)
             timestamp[0] = '\0';
         }
 
+        char pid[20] = {};
```

Comment:
buffer is size 20 because largest 64 bit number is 20 characters long

Reply:
Then it should be 21 - we need one more byte for terminating zero.